packages/eslint-config-sheriff/src/handpickedRules/getBaseEslintHandPickedRules.ts

```diff
@@ -1,5 +1,8 @@
-import { getIndexedBaseNoRestrictedSyntaxRules } from '../utils/getIndexedBaseNoRestrictedSyntaxRules';
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
```

Comment:
the inferred return type is very long and complex. Better to just have it inferred rather than written explicitly

Reply:
Isn't it just `TSESLint.FlatConfig.Rules`?

Reply:
i think it's more fine-grained (like with `as const`)

Reply:
The `as const` ensures you don't lose typing info, but instead of using satisfies, you can just annotate the return type, which should give the same result, but fix the linter error because it helps ts perf, right?

Reply:
but if i annotate the return type i lose the detailed typing info

Reply:
As much as I love detailed info, is there a reason you need it? Isn't exposing what specifically comes out of it just leaking internals and encouraging violating the layers of abstraction?